SalesforceFoundation/HEDAP
# Feature/xinnanli/primary faculty bug

# Critical Changes

# Changes
Avoid null pointer exception when assigning a course offering to a different primary faculty.
# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes
1. Create a course offering and fill in the primary faculty
2. Go to the primary faculty contact record page and delete the affiliation between the contact and the department
3. Assign the course offering to a different faculty
4. Make sure no null pointer exception is thrown and a new affiliation is created between new faculty and the department
